activerecord/lib/active_record/connection_adapters/postgresql/schema_creation.rb

```diff
@@ -84,6 +93,10 @@ def add_column_options!(sql, options)
             super
           end
 
+          def quoted_include_columns(o)
+            String === o ? o : quoted_include_columns_for_index(o)
+          end
```

Comment:
Isn't existing `quoted_columns` method already covers this?

Reply:
It is similar, but `quoted_columns` is fairly specific to the behavior surrounding the column options. It makes direct use of the `index.columns` and `index.column_options`, whereas for the `include` columns, we just need `index.include`. I considered refactoring `quoted_columns` to be a bit more generic, but the method it calls: `quoted_include_columns_for_index` also behaves slightly differently in how it handles the incoming params.

Reply:
👍 Yes, missed that it uses other index-specific settings.